src/app/components/ScoreTable.js

```diff
@@ -0,0 +1,57 @@
+import ef from '../utils/elementFactory';
```

Comment:
I am not very much in favour of shortening the names. I think it would be better to leave the full name

Reply:
Yeah, I would agree with if it would be used once/twice. In this case, when the function is quite widely used, in my opinion the short name makes the code clearer.